From 057d69729e52125219e3a94a71a9c58a57193bd8 Mon Sep 17 00:00:00 2001
From: Ankit Agarwal <ankit.agarwal@intel.com>
Date: Tue, 18 Jun 2024 10:12:52 +0530
Subject: [PATCH] Fixed build error in mediasdk_c2 component.

Observed build error due to missing assert header.
Added assert header.

Tests: Prepared EB, there was no build error.

Tracked-On: NA
Signed-off-by: Ankit Agarwal <ankit.agarwal@intel.com>
---
 c2_components/src/mfx_c2_component.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/c2_components/src/mfx_c2_component.cpp b/c2_components/src/mfx_c2_component.cpp
index 0980177..6cdd74e 100755
--- a/c2_components/src/mfx_c2_component.cpp
+++ b/c2_components/src/mfx_c2_component.cpp
@@ -23,6 +23,7 @@
 #include "mfx_c2_debug.h"
 #include "mfx_c2_components_registry.h"
 #include "mfx_c2_components_monitor.h"
+#include <assert.h>
 
 using namespace android;
 
-- 
2.34.1

